php/class-post-type.php

```diff
@@ -341,11 +353,15 @@ public function render_data_metabox( $post ) {
 
 			$frontend_view_url = get_permalink( $post->ID );
 			echo sprintf(
-				'<a href="%s" class="button button-secondary">%s</a>',
+				'<a href="%s" class="button button-secondary">%s</a> ',
```

Comment:
Just being consistent with https://github.com/xwp/wp-customize-snapshots/blob/0c20ece186044ad94a0d381c394c15d4030b83c4/php/class-post-type.php#L349

Should I remove space between both button and add CSS?